java/sidecar-common/src/main/java/com/instaclustr/sidecar/operations/OperationFactory.java

```diff
@@ -1,7 +1,6 @@
 package com.instaclustr.sidecar.operations;
 
+@FunctionalInterface
```

Comment:
Why does this need to be marked as a functional interface? It's not used as a lambda anywhere AFAIK. Its a Guice AssistedInject factory...